src/ServiceProvider/ApiClientProvider.php

```diff
@@ -48,6 +48,7 @@ public function provide(ServiceContainer $container)
                 ClientConfig::BASE_URI => $container->getParameter('ps_accounts.accounts_api_url'),
                 ClientConfig::SSL_CHECK => $container->getParameter('ps_accounts.check_api_ssl_cert'),
                 ClientConfig::HEADERS => [
+                    // FIXME: triggers SET cookie on AccountsService instantiation
                     'X-Anonymous-ID' => $container->get(AnalyticsService::class)->getAnonymousId(),
```

Comment:
@PrestaShopCorp/squad-account This is where cookie is set every time AccountsService is instantiated

Reply:
@antoinemetifeu a-t-on vraiment besoin de faire matcher le `x-anonymous-id` avec celui de Segment ?